sw/device/lib/handler.h

```diff
@@ -16,4 +16,70 @@ typedef enum exc_id {
   kIdMax = 31
 } exc_id_t;
 
+// Ibex only supports vectored interrupts, but the handler will not appear in
```

Comment:
Seeing as this file is somewhat Ibex-specific, should we consider naming it "ibex_irq_handlers.h" or similar?

Reply:
Ok, I'll document this further as it relates to the RISC-V spec, and change back to `aligned(4)` if we want this file to not be ibex-specific.